package.json

```diff
@@ -30,81 +31,82 @@
     ]
   },
   "dependencies": {
-    "@react-native-community/async-storage": "1.6.2",
-    "babel-preset-es2015": "6.24.1",
-    "base-64": "0.1.0",
-    "fast-password-entropy": "1.1.0",
-    "lodash": "4.17.15",
+    "@react-native-async-storage/async-storage": "1.15.5",
+    "@react-native-community/masked-view": "0.1.11",
+    "@sentry/react-native": "^2.6.0",
```

Comment:
Let's fix all versions, just in case...

Reply:
Fixed versions here a85ca06dd674b81c6ca5c0478882bbb744c79b8c